js/src/views/edit-attribute-field-attachment.js

```diff
@@ -29,10 +29,6 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 		if ( editAttributeFieldAttachment.getFromCache( id ) ) {
 			self._renderPreview( editAttributeFieldAttachment.getFromCache( id ) );
 			return;
-
-			// Call the updateValue() function, to trigger any listeners
-			// hooked on it.
-			self.triggerCallbacks();
```

Comment:
Nice when linting actually helps you find bugs :)

This shouldn't be removed, but it does need to be moved before the return statement. Otherwise, callback events won't be triggered when changing an attachment to one that's already been previously selected.